Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/ServiceCollectionExtensions.cs

```diff
@@ -35,8 +37,23 @@ public static class ServiceCollectionExtensions
         /// </summary>
         /// <param name="services"></param>
         /// <param name="eventSource"></param>
+        /// <param name="serializer"></param>
         /// <returns></returns>
         public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection services, LambdaEventSource eventSource)
+        {
+            // Not running in Lambda so exit and let Kestrel be the web server
+            return services.AddAWSLambdaHosting(eventSource, new DefaultLambdaJsonSerializer());
+        }
+
+        /// <summary>
+        /// Add the ability to run the ASP.NET Core Lambda function in AWS Lambda. If the project is not running in Lambda 
+        /// this method will do nothing allowing the normal Kestrel webserver to host the application.
+        /// </summary>
+        /// <param name="services"></param>
+        /// <param name="eventSource"></param>
+        /// <param name="serializer"></param>
+        /// <returns></returns>
+        public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection services, LambdaEventSource eventSource, ILambdaSerializer serializer)
```

Comment:
Instead of using a static place holder I think it would be more idiomatic to have this method that takes in an action callback that can be used for configuing.

Something like 
```
public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection services, LambdaEventSource eventSource, ILambdaSerializer serializer, Action<HostingOptions> configure = null)
{
    var options = new HostingOptions();
     if(configure != null) configure.Invoke(options);

    ...

    var serializer = options.Serializer ?? new DefaultLambdaJsonSerializer();

   ...
}
```

Reply:
@normj code updated, I've added an integration test as well. I think I understood the integration test framework correctly and they pass on my machine. Let me know if you want any more changes.